doc/conf.py

```diff
@@ -101,11 +101,22 @@
 latex_use_xindy = True
 
 linkcheck_timeout = 5
+linkcheck_ignore = [
+    r'^contents\.html$',  # extra generated page
+    '^\.\./contents\.html$',
+    re.escape('https://gitlab.com/projects/new'),  # requires sign-in
+    re.escape('https://web.libera.chat/?channel=#sphinx-doc'),
```

Comment:
@picnixz Could you double-check that these are correctly filtering these links?  When I ran this locally I saw a `broken` result for the `#sphinx-doc` channel when I expected an `ignored` result.

(sorry, late reply)

Reply:
It's weird because I did not have the issue on my side. I'll try it again in a few moment

Reply:
Ok, great - thanks; confirmed here too.

```
(           index: line    4) -ignored- https://web.libera.chat/?channel=#sphinx-doc 
```

Reply:
(reason for the filename diff: I moved it temporarily to a near-empty `index` file)